roles/build-release/tasks/build.yaml

```diff
@@ -58,6 +63,9 @@
   args:
     chdir: "{{ playbook_dir | dirname }}"
     creates: "{{ antsibull_data_dir }}/{{ _deps_file }}"
+  when: >-
+    not antsibull_skip_prepare
```

Comment:
Why have a separate variable if the playbook already skips the step when the deps file is missing?

Reply:
Oh, I completely missed the `creates`...

I guess this part isn't necessary then.